doc/api/stream.md

```diff
@@ -1143,6 +1143,15 @@ added: v12.3.0
 
 Getter for the property `objectMode` of a given `Readable` stream.
 
+##### readable.readableListening
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+Getter for the property `readableListening` of a given `Readable` stream.
```

Comment:
This would need to be better specified. What does this property mean?

Reply:
Yes, I can add more details on it indicating its meaning. Does this sound appropriate:

This property indicates if any active listeners exist for the given `Readable` stream.